demoScripts/aksQuickStart.md

```diff
@@ -0,0 +1,426 @@
+
+[!INCLUDE [quickstarts-free-trial-note](../../../includes/quickstarts-free-trial-note.md)]
+
+To learn more about creating a Windows Server node pool, see [Create an AKS cluster that supports Windows Server containers](quick-windows-container-deploy-cli.md).
+
+[!INCLUDE [azure-cli-prepare-your-environment.md](~/articles/reusable-content/azure-cli/azure-cli-prepare-your-environment.md)]
+
+- This article requires version 2.0.64 or later of the Azure CLI. If using Azure Cloud Shell, the latest version is already installed.
+
+- The identity you are using to create your cluster has the appropriate minimum permissions. For more details on access and identity for AKS, see [Access and identity options for Azure Kubernetes Service (AKS)][aks-identity-concepts].
+
+- If you have multiple Azure subscriptions, select the appropriate subscription ID in which the resources should be billed using the
+[az account][az-account] command.
+
+- Verify *Microsoft.OperationsManagement* and *Microsoft.OperationalInsights* providers are registered on your subscription. These are Azure resource providers required to support [Container insights][azure-monitor-containers]. To check the registration status, run the following commands:
+
+    ```azurecli
+    az provider show -n Microsoft.OperationsManagement -o table
+    az provider show -n Microsoft.OperationalInsights -o table
+    ```
+
+    If they are not registered, register *Microsoft.OperationsManagement* and *Microsoft.OperationalInsights* using the following commands:
+
+    ```azurecli
+    az provider register --namespace Microsoft.OperationsManagement
+    az provider register --namespace Microsoft.OperationalInsights
+    ```
+
+> [!NOTE]
+> Run the commands with administrative privileges if you plan to run the commands in this quickstart locally instead of in Azure Cloud Shell.
+
+## Define Environment Variables
+```azurecli-interactive
+export RESOURCE_GROUP_NAME=myResourceGroup4
+export RESOURCE_LOCATION=eastus
+export AKS_CLUSTER_NAME=myAKSCluster
+```
+
+## Create a resource group
+
+An [Azure resource group][azure-resource-group] is a logical group in which Azure resources are deployed and managed. When you create a resource group, you are prompted to specify a location. This location is:
+
+* The storage location of your resource group metadata.
+* Where your resources will run in Azure if you don't specify another region during resource creation.
+
+The following example creates a resource group named *myResourceGroup* in the *eastus* location.
+
+Create a resource group using the [az group create][az-group-create] command.
+
+```azurecli-interactive
+az group create --name $RESOURCE_GROUP_NAME --location $RESOURCE_LOCATION
+```
+
+The following output example resembles successful creation of the resource group:
+<!--expected_similarity=0.2-->
```

Comment:
Ack on thinking about a best practice. Because of all the differences, these tend to come out lower than I would hope. For example, here is one for VM create that was just today which failed.
<img width="822" alt="image" src="https://user-images.githubusercontent.com/91570951/218205359-feccb05e-0330-4e4f-898e-266fbf01ef17.png">